frontend/VideoPlayer.tsx

```diff
@@ -87,8 +90,21 @@ function YouTubeKeyControl({ player }: { player: YouTubePlayer }) {
         }
       }
     }
+    async function handleKeyup(e: KeyboardEvent){
+      if ((e.target as any)?.tagName !== 'INPUT') {
```

Comment:
To prevent it being inadvertantly paused by pressing an unassigned key (300ms after an assigned key):
```suggestion
      if ((e.target as any)?.tagName !== 'INPUT' && timestamps?.[e.key]) {
```